waspc/test/Generator/FileDraft/CopyAndModifyTextFileDraftTest.hs

```diff
@@ -0,0 +1,29 @@
+module Generator.FileDraft.CopyAndModifyTextFileDraftTest where
+
+import Fixtures (systemSPRoot)
+import qualified Generator.MockWriteableMonad as Mock
+import qualified StrongPath as SP
+import Test.Tasty.Hspec
+import Wasp.Generator.FileDraft
+
+spec_CopyAndModifyTextFileDraft :: Spec
+spec_CopyAndModifyTextFileDraft = do
+  describe "write" $ do
+    it "Creates new file by copying existing text file and applying modifications to it" $ do
+      let mock = write dstDir fileDraft
+      let mockLogs = Mock.getMockLogs mock Mock.defaultMockConfig
+      Mock.createDirectoryIfMissing_calls mockLogs
+        `shouldBe` [(True, SP.toFilePath $ SP.parent expectedDstPath)]
+      Mock.readFileAsText_calls mockLogs
+        `shouldBe` [SP.toFilePath expectedSrcPath]
+      Mock.writeFileFromText_calls mockLogs
+        `shouldBe` [(SP.toFilePath expectedDstPath, "First line\nMock text file content")]
+  where
+    (dstDir, dstPath, srcPath) =
+      ( systemSPRoot SP.</> [SP.reldir|a/b|],
+        [SP.relfile|c/d/dst.txt|],
+        systemSPRoot SP.</> [SP.relfile|e/src.txt|]
+      )
+    fileDraft = createCopyAndModifyTextFileDraft dstPath srcPath ("First line\n" <>)
+    expectedSrcPath = srcPath
+    expectedDstPath = dstDir SP.</> dstPath
```

Comment:
This goes against of the convention of importing `(</>)` from StrongPath vs. using it `SP.</>`

Reply:
I removed `SP.` and imported SP stuff with list!
That said, I wouldn't say we have a strict convention there -> I think we do prefer somewhat importing it as a list of stuff vs qualified because we use it quite a lot in the codebase so it becomes boring to look at `SP.` all the time, but on the other hand it can be ok situationally to use `SP.` sometimes, it is up to the code author really I think.